_includes/work-at-triggerise.html

```diff
@@ -22,19 +22,12 @@
         </div>
       </div>
       <div class="uk-video pink uk-card-center uk-card-body">
-            <iframe id="yTPlayer" src="https://www.youtube.com/embed/jxyPJiLL4Kw?autoplay=true&mute=1;
-                rel=0&amp;
-                modestbranding=1
-                &amp;playsinline=1" 
-                  width="100%" 
-                  height="695"
-                  frameborder="0" 
-                  uk-responsive 
-                  uk-video="autoplay: inview">
-            </iframe>
+        <div class="uk-child-width-auto@m">
+          <div class="iframe-container">
+            <iframe width="560" height="315" src="https://www.youtube.com/embed/jxyPJiLL4Kw?autoplay=true&mute=1; rel=0&amp;modestbranding=1" uk-video="autoplay: inview" frameborder="0" allow="accelerometer; autoplay; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Not super critical, but I would probably have similar to before and separate this into a few lines, just so it becomes more readable.

Reply:
Done.